src/Compilers/CSharp/Portable/Symbols/Tuples/TupleErrorFieldSymbol.cs

```diff
@@ -77,15 +97,17 @@ public override ImmutableArray<SyntaxReference> DeclaringSyntaxReferences
         {
             get
             {
-                return GetDeclaringSyntaxReferenceHelper<CSharpSyntaxNode>(_locations);
+                return _isImplicitlyDeclared ?
+                    ImmutableArray<SyntaxReference>.Empty :
+                    GetDeclaringSyntaxReferenceHelper<CSharpSyntaxNode>(_locations);
```

Comment:
For implicitly declared symbols (like property accessors) they point to place that "made them to be". In a case of   (int, int) , the location for "Item1"  is the first argument. 
It is not, however, its DeclaringSyntaxReference. Being an implicitly declared symbol, Item1 is not supposed to have those.

---
In reply to: [78256723](https://github.com/dotnet/roslyn/pull/13713#discussion_r78256723) [](ancestors = 78256723)